### DISCUSSION TITLE
API?

### DISCUSSION THREAD

**Discussion by prismspecs** [yownas/seed_travel#58] [General]
Is it possible to run this via API call?

**Comment by yownas** [comment]
Kind of, it should be possible to start the generation of the video but it won't be returned via the api call.

The main issue is that you can't return something that isn't an image (or list of images), or the webui will complain. (Or, rather the gradio Gallery box.) I don't know if there is an easy way to get around it.

**Reply by prismspecs** [reply]
Thanks for your reply! I managed to hack something together for my purposes-- good to know about not being able to return non-images, that helps me to understand why I couldn't get it to work!
